### PR TITLE
feat: release 3.9.0

### PR DESCRIPTION
**Pull request by shreemaan-abhishek**
### Description

release v3.9

Fixes # (issue)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -75,11 +76,52 @@ title: Changelog
...
+
+### Core:
+
+- :sunrise: Support HTTP3/QUIC
+  - [#10989](https://github.com/apache/apisix/pull/10989)
+  - [#11010](https://github.com/apache/apisix/pull/11010)
+  - [#11027](https://github.com/apache/apisix/pull/11027)
+- :sunrise: add plugins/reload to control api [#10905](https://github.com/apache/apisix/pull/10905)
+
+### Plugin:
```
Comment (kayx23) [inline CHANGELOG.md:94]:
```suggestion
### Plugins
```

Comment (kayx23) [reply CHANGELOG.md:94]: remove `:` from all sectional titles.

File: CHANGELOG.md [hunk 2]
```diff
@@ -75,11 +76,52 @@ title: Changelog
...
+- :sunrise: allow configuring allow-headers in grpc-web plugin [#10904](https://github.com/apache/apisix/pull/10904)
+- :sunrise: Add forward-auth plugin exception configuration status_on_error [#10898](https://github.com/apache/apisix/pull/10898)
+- :sunrise: add option to include request body and response body in log util [#10888](https://github.com/apache/apisix/pull/10888)
+- :sunrise: support compressed responses in loggers [#10884](https://github.com/apache/apisix/pull/10884)
+- :sunrise: add http-dubbo plugin [#10703](https://github.com/apache/apisix/pull/10703)
+- :sunrise: support built-in variables in response_headers in mocking plugin [#10872](https://github.com/apache/apisix/pull/10872)
+- :sunrise: support other data formats without warnings [#10862](https://github.com/apache/apisix/pull/10862)
+- :sunrise: add ocsp-stapling plugin [#10817](https://github.com/apache/apisix/pull/10817)
+
+### Bugfixes:
```
Comment (kayx23) [inline CHANGELOG.md:110]:
```suggestion
### Bug Fixes
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -75,11 +76,52 @@ title: Changelog
...
+
+### Core
+
+- :sunrise: Support HTTP3/QUIC
+  - [#10989](https://github.com/apache/apisix/pull/10989)
+  - [#11010](https://github.com/apache/apisix/pull/11010)
+  - [#11027](https://github.com/apache/apisix/pull/11027)
+- :sunrise: add plugins/reload to control api [#10905](https://github.com/apache/apisix/pull/10905)
+
+### Plugin
```
Comment (AlinsRan) [inline CHANGELOG.md:94]:
```suggestion
### Plugins
```

**Review by AlinsRan — APPROVED** [review]

**Review by Revolyssup — APPROVED** [review]

**Pull request merged**